README.ja.md

```diff
@@ -23,7 +23,7 @@ uroboroSQLは主にSQL中心の設計コンセプトを採用しています。J
 <dependency>
     <groupId>jp.co.future</groupId>
     <artifactId>uroborosql</artifactId>
-    <version>0.26.3</version>
+    <version>1.0.0</version>
```

Comment:
When merging this PR, let's set it to 1.0.0-SNAPSHOT.

Reply:
OK